recipe/meta.yaml

```diff
@@ -1,7 +1,7 @@
 {% set name = "pre-commit" %}
-{% set version = "1.12.0" %}
+{% set version = "1.18.2" %}
 {% set hash_type = "sha256" %}
-{% set hash = "81fe7b1b0cba93e7d7aa3cfbfd55f5280d3639e4fcb14bf9d1b95ab35f41cf0d" %}
```

Comment:
the deps have drifted a bit -- looks like this got copied from an old version of this recipe:

https://github.com/pre-commit/pre-commit/blob/7f900395ec8fa2de7962694e11a206af33dc9fcd/setup.cfg#L27-L37

Reply:
What do you mean, the `run` section? It seems fine to me:

https://github.com/pre-commit/pre-commit/blob/7f900395ec8fa2de7962694e11a206af33dc9fcd/setup.cfg#L26-L37

Reply:
I guess the only difference is pre-commit has `cfgv>=2`, conda recipe has `>= 1`

Reply:
Oh 🤦‍♂ 

Fixed, thanks for catching it. 👍

Reply:
o7 no problem

also looks like `deadsnakes-issues-bot` got dropped in the move

Reply:
> also looks like deadsnakes-issues-bot got dropped in the move

I'm not aware of that bot, can you clarify please?

Reply:
oh, it's my second user account I use for low-priority notifications: https://github.com/conda-forge/pre_commit-feedstock/blob/23fecf53fe707713c49dae3ea216ec234ac255bc/recipe/meta.yaml#L66

Reply:
Oh OK, my bad, fixed. 👍